pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -8,7 +8,7 @@ help() {
 Welcome to EVE!
   commands: enter [service] [command (assumed sh)]
             enter-user-app <service>
-            exec [service] command
+            exec [--fork] [service] command
```

Comment:
I see option '--fork' but not '--no-fork' as commit msg says.

Reply:
Fixed